cmd/manager/main.go

```diff
@@ -40,6 +40,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
+// The default durations for the leader election operations.
+var (
+	leaseDuration = 120 * time.Second
+	renewDeadline = 110 * time.Second
+	retryPeriod   = 20 * time.Second
```

Comment:
Why is this 20 seconds?  Aren't we doing 90 elsewhere?

Reply:
hmm, i wonder if that is just a copy-paste error on my part. i'm gonna put a hold on this while i look into it.
/hold

Reply:
seems like it was a c-p error, i need to update this and the other PRs. thanks for the catch Mike!

Reply:
@michaelgugino @elmiko This copy-paste error appeared only because `20s` for retry period is more than enough. The typical number for lease-refresh events for other namespaces is between `18` and `56`. Our number was `268` for the default `retryPeriod` of `2s`. Now it is `20s` - 10 times more, so we should have `10` times less lease-refresh events, bringing us on the same level as other operators (`~27`). With `90s` `retryPeriod` I observed an uncomfortable number of CI failures due to flakes in overall e2e tests, assuming first attempt to acquire lease that's why `20s` is now set. Feel free to proceed with `90s` if you don't observe a similar issue.

Reply:
hmm, that's good info @Danil-Grigorev . i didn't realize about the flakes, i thought we were staying with 90s. i'm fine to revert if 20s is what we really want here.

Reply:
i'd like to figure this out before we merge though, so i'm gonna put the hold back on
/hold